apps/core/src/utils/getAmount.ts

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import {
+    type SuiTransactionResponse,
+    getTotalGasUsed,
+    SUI_TYPE_ARG,
+} from '@mysten/sui.js';
+
+// Get the amount from events and transfer data
+export function getAmount(txnData: SuiTransactionResponse) {
```

Comment:
This function name could probably be tweaked so that it's more obvious Also, do you think it's useful enough to move to the main SDK?